src/pages/billing/Project.js

```diff
@@ -251,7 +251,10 @@ const ProjectDetail = ({ authorizeAndLoadAccounts, billingAccounts, billingProje
   }
 
   const spendReportKey = 'spend report'
-  const maybeLoadTotalCost = reportErrorAndRethrow('Unable to retrieve spend report data')(async () => {
+  const maybeLoadTotalCost = _.flow(
+    reportErrorAndRethrow('Unable to retrieve spend report data'),
+    Utils.withBusyState(setUpdating)
+  )(async () => {
     if (!updatingTotalCost && totalCost === null && tab === spendReportKey) {
```

Comment:
One other tricky thing going on here is that if the ajax call to get the spend report throws an error, I don't want to keep calling it and get a stack of error notifications…. the call is expected to fail if billing permission information is not set up. This is why I'm managing `updatingTotalCost` here myself, so it remains `true` if the report errors.

We will probably want to handle this more elegantly before the feature is released (and show explanatory text vs. an error), but we are still early in development of this report and iteratively building up the UI.